docs/README.md

```diff
@@ -46,6 +47,43 @@ The `trustpilot` app also enables Truspilot's widgets to be added to your store
 | ------------------- | ------------------- | --------------------------------------- |
 | `shop-review-badge` | `trustpilot-widget` | Adds a Trustpilot widget in your store. |
 
+
+## Block without badge
+
+To use widget without `shop-review-badge` to landings for example you can use `trustpilot-widget`, you need add to your manifest this 
+
+```diff
+ "peerDependencies": {
++   "vtex.trustpilot": "1.x"
+ }
+```
+## `trustpilot-widget` as props
+
+```json
+{
+   "trustpilot-widget#home":{
+    "title":"Widget TrustPilot",
+    "props":{
+      "width":"100%",
+      "height":"240px",
+      "theme":"light",
+      "stars":"4,5",
+      "businessUnitId":"597a23fwffdsfe05a793fe",
+      "templateId": "5419b6a8b0d04a076446a9ad",
+    }
+  }
+}
+```
+
+| Prop name |   Type   |                         Description                         | Default value |
+| :-------: | :------: | :---------------------------------------------------------: | :-----------: |
+|  `width`  | `string` | Change width of template |  `100%`   |
+|  `height`  | `string` | Change height of template |  `240px`   |
+|  `theme`  | `string` | Change theme of template `light` or `dark` |  `light`   |
+|  `stars`  | `string` | Change stars stock of template |  `4,5`   |
+|  `businessUnitId`  | `string` | Set your business code  |  null  |
```

Comment:
Could you update the default values here to `undefined`?

Reply:
fixed

Reply:
fixed